torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -718,7 +740,9 @@ def crop_bounding_box(
     format: features.BoundingBoxFormat,
     top: int,
     left: int,
-) -> torch.Tensor:
+    height: int,
+    width: int,
```

Comment:
This is a little awkward: we only pass `height` and `width` here to return them unchanged. Thus, although the `image_size` is "decomposed" here, this is mostly belongs in the category of kernels that shouldn't return the `image_size` since it is not changed inside the kernel.

However, `crop_bounding_box` was an outlier in the first place, since all other `crop_*` kernels take the `height` and `width` parameter:

https://github.com/pytorch/vision/blob/c4c28dff8a68fae20eb7f2a82051ef7a388353bd/torchvision/transforms/functional_tensor.py#L132

https://github.com/pytorch/vision/blob/7046e56fe4370e94339b3e8b6fd011e285294a3a/torchvision/prototype/transforms/functional/_geometry.py#L735

https://github.com/pytorch/vision/blob/7046e56fe4370e94339b3e8b6fd011e285294a3a/torchvision/prototype/transforms/functional/_geometry.py#L739

Thus, with this change we are actually aligning the kernels although it is a little awkward for the bounding box one. No strong opinion here.

Reply:
Good call. No strong opinions on my side.